<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite skipped="0" hostname="AV-PN-NEHAP" name="TestCases.Regression.ThirdPartyAddressValidation" tests="8" failures="4" timestamp="07 Jan 2021 10:00:38 GMT" time="1576.726" errors="0">
  <testcase name="TestCase162_USAddressValidated" time="155.627" classname="TestCases.Regression.ThirdPartyAddressValidation"/>
  <testcase name="TestCase163_CANAddressValidated" time="157.953" classname="TestCases.Regression.ThirdPartyAddressValidation"/>
  <testcase name="TestCase164_MilitaryAddressValidated" time="157.995" classname="TestCases.Regression.ThirdPartyAddressValidation"/>
  <testcase name="TestCase165_NonUS_NonCANAddress" time="219.447" classname="TestCases.Regression.ThirdPartyAddressValidation">
    <failure type="java.lang.AssertionError" message="The following asserts failed:
	Expected:Error: This country is not supported for address validation. Only US and Canadian addresses can be validated at this time.	And Actual:Line 1: UNIT 23152, City: APO, State/Region: AE, Postal Code: 09067-3152, Country: US expected [true] but found [false],
	Expected:Error: This country is not supported for address validation. Only US and Canadian addresses can be validated at this time.	And Actual:Line 1: UNIT 23152, City: APO, State/Region: AE, Postal Code: 09067-3152, Country: US expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: The following asserts failed:
Expected:Error: This country is not supported for address validation. Only US and Canadian addresses can be validated at this time.And Actual:Line 1: UNIT 23152, City: APO, State/Region: AE, Postal Code: 09067-3152, Country: US expected [true] but found [false],
Expected:Error: This country is not supported for address validation. Only US and Canadian addresses can be validated at this time.And Actual:Line 1: UNIT 23152, City: APO, State/Region: AE, Postal Code: 09067-3152, Country: US expected [true] but found [false]
at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:43)
at TestCases.Regression.ThirdPartyAddressValidation.addressValidationforInvalidAddress(ThirdPartyAddressValidation.java:193)
at TestCases.Regression.ThirdPartyAddressValidation.TestCase165_NonUS_NonCANAddress(ThirdPartyAddressValidation.java:62)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- TestCase165_NonUS_NonCANAddress -->
  <testcase name="TestCase166_AddressOutOfRange" time="221.119" classname="TestCases.Regression.ThirdPartyAddressValidation">
    <failure type="java.lang.AssertionError" message="The following asserts failed:
	Expected:Error: The address was found but the street number in the input address was not between the low and high range of the post office database.	And Actual:Error: This country is not supported for address validation. Only US and Canadian addresses can be validated at this time. expected [true] but found [false],
	Expected:Error: The address was found but the street number in the input address was not between the low and high range of the post office database.	And Actual:Error: This country is not supported for address validation. Only US and Canadian addresses can be validated at this time. expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: The following asserts failed:
Expected:Error: The address was found but the street number in the input address was not between the low and high range of the post office database.And Actual:Error: This country is not supported for address validation. Only US and Canadian addresses can be validated at this time. expected [true] but found [false],
Expected:Error: The address was found but the street number in the input address was not between the low and high range of the post office database.And Actual:Error: This country is not supported for address validation. Only US and Canadian addresses can be validated at this time. expected [true] but found [false]
at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:43)
at TestCases.Regression.ThirdPartyAddressValidation.addressValidationforInvalidAddress(ThirdPartyAddressValidation.java:193)
at TestCases.Regression.ThirdPartyAddressValidation.TestCase166_AddressOutOfRange(ThirdPartyAddressValidation.java:67)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- TestCase166_AddressOutOfRange -->
  <testcase name="TestCase167_UnknownStreet" time="220.472" classname="TestCases.Regression.ThirdPartyAddressValidation">
    <failure type="java.lang.AssertionError" message="The following asserts failed:
	Expected:Error: An exact street name match could not be found and phonetically matching the street name resulted in either no matches or matches to more than one street name.	And Actual:Error: The address was found but the street number in the input address was not between the low and high range of the post office database. expected [true] but found [false],
	Expected:Error: An exact street name match could not be found and phonetically matching the street name resulted in either no matches or matches to more than one street name.	And Actual:Error: The address was found but the street number in the input address was not between the low and high range of the post office database. expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: The following asserts failed:
Expected:Error: An exact street name match could not be found and phonetically matching the street name resulted in either no matches or matches to more than one street name.And Actual:Error: The address was found but the street number in the input address was not between the low and high range of the post office database. expected [true] but found [false],
Expected:Error: An exact street name match could not be found and phonetically matching the street name resulted in either no matches or matches to more than one street name.And Actual:Error: The address was found but the street number in the input address was not between the low and high range of the post office database. expected [true] but found [false]
at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:43)
at TestCases.Regression.ThirdPartyAddressValidation.addressValidationforInvalidAddress(ThirdPartyAddressValidation.java:193)
at TestCases.Regression.ThirdPartyAddressValidation.TestCase167_UnknownStreet(ThirdPartyAddressValidation.java:72)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- TestCase167_UnknownStreet -->
  <testcase name="TestCase168_NonDeliverableAddress" time="223.866" classname="TestCases.Regression.ThirdPartyAddressValidation">
    <failure type="java.lang.AssertionError" message="The following asserts failed:
	Expected:Error: Address cannot be geocoded.	And Actual:Error: An exact street name match could not be found and phonetically matching the street name resulted in either no matches or matches to more than one street name. expected [true] but found [false],
	Expected:Error: Address cannot be geocoded.	And Actual:Error: An exact street name match could not be found and phonetically matching the street name resulted in either no matches or matches to more than one street name. expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: The following asserts failed:
Expected:Error: Address cannot be geocoded.And Actual:Error: An exact street name match could not be found and phonetically matching the street name resulted in either no matches or matches to more than one street name. expected [true] but found [false],
Expected:Error: Address cannot be geocoded.And Actual:Error: An exact street name match could not be found and phonetically matching the street name resulted in either no matches or matches to more than one street name. expected [true] but found [false]
at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:43)
at TestCases.Regression.ThirdPartyAddressValidation.addressValidationforInvalidAddress(ThirdPartyAddressValidation.java:193)
at TestCases.Regression.ThirdPartyAddressValidation.TestCase168_NonDeliverableAddress(ThirdPartyAddressValidation.java:77)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- TestCase168_NonDeliverableAddress -->
  <testcase name="TestCase169_CityError" time="220.247" classname="TestCases.Regression.ThirdPartyAddressValidation"/>
</testsuite> <!-- TestCases.Regression.ThirdPartyAddressValidation -->
